concourse/execute-test-runs/script.bash

```diff
@@ -5,20 +5,22 @@ BASE_DIR=$(pwd)
 
 cd rex-web
 
-for MILESTONE_NUMBER in $(ls $BASE_DIR/unified-issues-milestones)
+for MILESTONE_NUMBER in "$BASE_DIR"/unified-issues-milestones/*
```

Comment:
@philschatz this internal variable is different after this change. previously it was just the directory name, with the glob now its getting the full path.

why did you change this?

Reply:
Thanks for testing (or checking the concourse build)! It is probably wrong but I changed it because of the shellcheck message [SC2045: Iterating over ls output is fragile. Use globs](https://github.com/koalaman/shellcheck/wiki/SC2045)

I can add a `# shellcheck disable=SC2045` if there is no good way to write it properly.

https://github.com/koalaman/shellcheck/wiki/Ignore#ignoring-one-specific-instance-in-a-file

Reply:
is there a way to get the glob to only return the directory name instead of the whole path?

Reply:
hmm, my shell-fu is not great but `basename ${MILESTONE_NUMBER}` might work. How can I test it?

Reply:
https://stackoverflow.com/questions/2664740/extract-file-basename-without-path-and-extension-in-bash might also be relevant to our interests. its hard to test, if you push to this branch concourse will run and "run test plans" should turn green here https://concourse.openstax.org/teams/Dev/pipelines/unified

there is a way to test the job before pushing with `fly execute` but i haven't totally figured out the workflow especially when you have lots of inputs

Reply:
Thanks! I'll try to make the fixes now that I have a way to verify they work (I forgot that concourse was already running 😊 ).

Also, I got feedback from https://stackoverflow.com/users/26428/dennis-williamson ( @dtwilliamson ) a while back that local variables should be `under_snake` case so they are visually distinct from `ENVIRONMENT_VARS`

Reply:
that makes sense on variable naming